include/tunable_static_tf_broadcaster.h

```diff
@@ -0,0 +1,33 @@
+#ifndef TUNABLE_STATIC_TF_BROADCASTER_H_
+#define TUNABLE_STATIC_TF_BROADCASTER_H_
+
+#include <memory>
+#include <tf2_ros/transform_broadcaster.h>
+#include <geometry_msgs/TransformStamped.h>
+#include <dynamic_reconfigure/server.h>
+#include "tunable_static_tf_broadcaster/TfConfig.h"
+
+namespace tunable_static_tf_broadcaster
+{
+    class TunableStaticTfBroadcaster
+    {
+    public:
+        TunableStaticTfBroadcaster();
+        ~TunableStaticTfBroadcaster();
+        void run();
+
+    private:
+        ros::NodeHandle private_nh_;
+        tf2_ros::TransformBroadcaster broadcaster_;
+        geometry_msgs::TransformStamped transform_;
+        std::shared_ptr<dynamic_reconfigure::Server<TfConfig> > dynamic_reconfigure_server_;
```

Comment:
In ubuntu16.04, ros kinetic environment, an error occurred when building as it is.
Please describe options of c++11 or higher in CMakeList.

Reply:
I fixed that.